### ISSUE TITLE
Rooms get created and destroyed every 10 seconds

### ISSUE THREAD

**Issue by licaon-kter** [jitsi/jicofo#470]
This is the health check?

There isn't any other way to check like [XEP-0199: XMPP Ping](https://xmpp.org/extensions/xep-0199.html) or something?

My ejabberd server and logs (and SD card) get constantly spammed with:
```
12:26:51.307 [info] <0.26548.0>@mod_muc_room:init:281 Created MUC room org.jitsi.jicofo.health.health-732e0266d682b3e9@conference.meet.domain.tld by focus@domain.tld/focus118982299115267
12:26:56.623 [info] <0.26547.0>@mod_muc_room:close_room_if_temporary_and_empty:1320 Destroyed MUC room org.jitsi.jicofo.health.health-ae759bd65ed038da@conference.meet.domain.tld because it's temporary and empty
12:26:56.630 [info] <0.26547.0>@mod_muc_room:terminate:874 Stopping MUC room org.jitsi.jicofo.health.health-ae759bd65ed038da@conference.meet.domain.tld3112
```

**Comment by damencho** [comment]
It is not just connection, we are testing whether jicofo is not in a weird state or the xmpp server as we have been seeing such issues.

**Comment by damencho** [comment]
Please use the community forum for questions or problems before opening new issues.

**Comment by licaon-kter** [comment]
How can I disable this check?

> community forum 

No "login with Github/lab" ? Meh

**Comment by damencho** [comment]
You can use the forum with github account.

**Comment by damencho** [comment]
And I don't see a way to disable that at the moment.

**Comment by bgrozev** [comment]
@licaon-kter A patch to make this configurable would be welcome. You'd have to pick up an updated jicoco (with jitsi/jicoco#80) and modify [Health](https://github.com/jitsi/jicofo/blob/master/src/main/java/org/jitsi/jicofo/health/Health.java) to read configuration from a `ConfigurationService`

**Comment by vanitasvitae** [comment]
Just want to chime in to conform this issue.
JM just brought my ejabberd down with ~60000 MUC rooms and ~1.6m caps entries.

**Comment by bgrozev** [comment]
Working on a fix

**Comment by bgrozev** [comment]
Should be fixed in #494
